package.json

```diff
@@ -1,10 +1,14 @@
 {
   "name": "marigold",
-  "version": "0.1.0",
+  "version": "0.2.0",
   "description": "A set of common and reusable components and tools for building websites and applications at 18F.",
   "scripts": {
-    "build": "fractal build",
-    "start": "fractal start --sync"
+    "prestart": "npm run clean && npm run build",
+    "postinstall": "source env.sh",
```

Comment:
Whats this needed for? I'm super hesitant about these types of npm scripts (the automatic ones). Although because this is only making a dir it might be fine. My hesitation is based on when we used `prepublish` on WDS, which was then run when you ran `npm install wds` on a completely separate project. This was bad because it ran linting which required ruby which shouldn't have been a thing when you were just install WDS.

I guess my point is: the automatic npm commands get run when you might not expect sometimes.